pystac/serialization/common_properties.py

```diff
@@ -89,8 +89,9 @@ def merge_common_properties(
                 collection_props = collection["properties"]
         else:
             raise ValueError(
-                "{} is expected to be a Collection or "
-                "dict but is neither.".format(collection)
+                "{} is expected to be a Collection or " "dict but is neither.".format(
+                    collection
```

Comment:
It was this kind of change that made me think the line length had changed. But I think you are right that it it's just changing how they break.